builtins/util.m4

```diff
@@ -5686,6 +5686,15 @@ define void @__memset64(i8 * %dst, i8 %val, i64 %len) alwaysinline {
     ret void
 }
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; assume
+declare void @llvm.assume(i1)
+
+define void @__do_assume_uniform(i1 %test) alwaysinline {
```

Comment:
Do we need a mask argument?

Reply:
For uniform? I based it off assert implemetation. We just use the condition to make decision

```
declare i32 @puts(i8*)
declare void @abort() noreturn

define void @__do_assert_uniform(i8 *%str, i1 %test, <WIDTH x MASK> %mask) {
  br i1 %test, label %ok, label %fail

fail:
  %call = call i32 @puts(i8* %str)
  call void @abort() noreturn
  unreachable

ok:
  ret void
}
```

Reply:
yeah, this is correct.